src/main/java/com/example/myflightplanner/repository/FlightsRepository.java

```diff
@@ -0,0 +1,52 @@
+package com.example.myflightplanner.repository;
+
+import com.example.myflightplanner.models.Airport;
+import com.example.myflightplanner.models.Flight;
+import com.example.myflightplanner.models.PageResult;
+import com.example.myflightplanner.models.SearchFlight;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.server.ResponseStatusException;
+
+@Repository
+public class FlightsRepository {
+
+  List<Flight> flightList = new ArrayList<>();
+
+  public void clear() {
+    flightList.clear();
+  }
+
+
+  @Async
```

Comment:
Šī anotācija gan nepalīdz tikt galā ar multithread problēmām:
https://www.baeldung.com/spring-async
"..a method of a bean with @Async will make it execute in a separate thread. In other words, the caller will not wait for the completion of the called method."

Testi visu laiku iziet? Jo te pietiktu, ja metodei norādītu synchronized

Reply:
Es nezinu, jo es veicu async testu 20 reizes un tikai vienu reizi uzrādīju kļūdu, bet ar synchronized tikai 14 reizes veiksmīgi nokārtotas testi

Reply:
it ka pafiksēju concurrency issue, tgd 20 reizes testi bija ok